lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.blobs.get('<Container name>', '<Blob name>').create(file_path: '<file path>')
```

Comment:
So you are validating if blob exists and then calling .create method. It's a user doc and we want to tell the user that how can he upload a local file as a blob. Let's not do validation and keep it simple as follows:
```

new_blob = azure_storage_service.blobs.create(
  name: 'myblob', 
  container_name: 'mycontainer', 
  file_path: '<file path>')
```